modules/core/src/test/scala/io/janstenpickle/hotswapref/HotswapRefSpec.scala

```diff
@@ -48,11 +48,11 @@ class HotswapRefSpec extends AnyFlatSpec with Matchers with TestInstances {
       )
       _ <- Resource.eval(IO.sleep(1.second)) // give it a chance to update
       _ <- Resource.eval(hotswap.swap(Resource.pure(ref1)).start)
```

Comment:
The description of the test is misleading because there is no blocking on swap, otherwise the test wouldn't complete. The way `cats.effect.std.Hotswap` works is outlined in the scaladoc for the class, so the new resource is allocated _before_ the old one is released.